assets/js/util/tracking/createInitializeSnippet.js

```diff
@@ -56,7 +56,8 @@ export default function createInitializeSnippet( config, dataLayerTarget ) {
 			? userRoles.join( ',' )
 			: '';
 
-		dataLayerPush( 'js', new Date() );
+		// Don't use the getReferenceDate selector here since there is no need in it.
```

Comment:
The IB says this file is only used in tests, but that doesn't seem accurate (as it's used here: https://github.com/google/site-kit-wp/blob/774ea23c2471170c96898f11c1909dedf6bc5db3/assets/js/util/tracking/index.js#L50-L57).

But seems like the initialisation should match the real time it happened, so a comment explaining that is fine:

```suggestion
		// We should use the real time, and not the reference date, when
		// measuring when we initialized tracking.
```